src/index.html

```diff
@@ -11,8 +11,63 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with a .scss extension won't work directly in browsers. You should link to the compiled .css file instead.